doc: helm : tiny improvements and fixes (#3595)
operator-framework/operator-sdk: website/content/en/docs/building-operators/helm/tutorial.md

---
title: Tutorial for Helm-based Operators
linkTitle: Tutorial

> weight: 2

Reviewer: ```suggestion
weight: 200
```

Author: I changed to show the reference at the end.

> description: A simple set of instructions for an in-depth walkthough to demonstrates the basics of setting up and running a Helm-based operator. 

Reviewer: ```suggestion
description: An in-depth walkthough that demonstrates how to build and run a Helm-based operator. 
```
